src/scheduler/server_info.c

```diff
@@ -929,6 +931,15 @@ query_sched_obj(status *policy, struct batch_status *sched, server_info *sinfo)
 				conf.preempt_min_wt_used = 1;
 			else
 				conf.preempt_min_wt_used = 0;
+		} else if (!strcmp(attrp->name, ATTR_job_sort_formula)) {
+			if (conf.job_sort_formula != NULL)
+				free(conf.job_sort_formula);
+			conf.job_sort_formula = read_formula();
```

Comment:
If you are going to read this every cycle, leave it where it was (sinfo).  If you want to read it only when it changes, follow the code for command SCH_ATTRS_CONFIGURE and shove it in global space somewhere.

Reply:
In order to support backwards compatibility for server's job_sort_formula attribute, we need to read it every cycle. So I'll just leave it with sinfo, we can change this to follow the SCH_ATTRS_CONFIGURE path when we remove the server's JSF.